examples/step-85/doc/results.dox

```diff
@@ -0,0 +1,17 @@
+<h1>Results</h1>
+
+In 2D, the analytical solution for this problem is
+@f{align*}
+  u(x) = 2 - \| x \|^2, \qquad x \in \overline{\Omega}.
+@f}
+The numerical solution is shown in the below figure.
+The zero-contour of the level set function is shown as a white line.
+Since we use FENothing elements on the cells that are not in the active mesh,
+we see that the solution is zero there.
+On the intersected cells,
+we see that the numerical solution has a value also outside $\overline{\Omega}$.
+Note that this extension of the solution is artificial.
+The value of the numerical solution and thus also the error, $u_h - u$,
```

Comment:
Maybe you could extend the tutorial so that also outputs the L2 norm of the error? For that you could simply use `NonMatching::FEValues`. Also adding a small convergence table here would be nice. It is quite common in the tutorials!